src/shell/meanbee/configpoweredcss/regenerate.php

```diff
@@ -0,0 +1,48 @@
+<?php
+require_once __DIR__ . '/../../abstract.php';
+
+class Meanbee_Configpoweredcss_Regenerate extends Mage_Shell_Abstract
+{
+
+    /**
+     * Run script
+     *
+     */
+    public function run()
+    {
+
+        /**
+         * Shell script regenerate the configpowered css for all stores
+         */
+
+        try {
+
+            /** @var Meanbee_ConfigPoweredCss_Model_Css $css */
+            $css = Mage::getModel('meanbee_configpoweredcss/css');
+
+            $stores = Mage::app()->getStores();
+            foreach ($stores as $storeId => $store) {
+                try {
+                    Mage::app()->setCurrentStore($storeId);
+                    $css->publish($storeId);
+                } catch (Exception $e) {
+                    Mage::logException($e);
+                    return;
```

Comment:
This will stop on the first exception, but exit with no message to the user and a successful error code. We should show an error and use an error code to exit.